.github/workflows/yamllint.yaml

```
@@ -21,6 +21,8 @@ jobs:
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
       - uses: azure/setup-kubectl@776406bce94f63e41d621b960d78ee25c8b76ede # v4
       - uses: azure/setup-helm@1a275c3b69536ee54be43f2070a358922e12c8d4 # v4.3.1
+        with:
+          version: v3.7.2
```

```suggestion
          version: v3.18.4
```

That'll at least match what we're using inside of ArgoCD

---

Moved that fix to: https://github.com/rackerlabs/understack/pull/1396